### ISSUE TITLE
harmony-api not accepting publish requests?

### ISSUE THREAD

**Issue by clemon79** [maddox/harmony-api#51]
The following should turn off my system:
```
mosquitto_pub -d -u <username> -P <password> -t harmony-api/hubs/freds-harmony-hub/activities/watch-xbox-one/command -m off
```
It returns this:
```
Client mosqpub/2200-raspberryp sending CONNECT
Client mosqpub/2200-raspberryp received CONNACK
Client mosqpub/2200-raspberryp sending PUBLISH (d0, q0, r0, m1, 'harmony-api/hubs/freds-harmony-hub/activities/watch-xbox-one/command', ... (3 bytes))
Client mosqpub/2200-raspberryp sending DISCONNECT
```
But nothing happens.

This:
```
mosquitto_sub -v -h 'localhost' -u '<username>' -P '<password>' -t 'harmony-api/#'
```
...returns the list of activities, so watching seems to be working, just not publishing changes.

The switch as defined in MQTT is correctly reflecting the state but will not execute a command when manually flipped.

**Comment by mattsch** [comment]
I'm seeing the same behavior, rolled back again to fix things.

**Comment by clemon79** [comment]
Yeah, I basically backed out altogether, now that all of the functionality I need is baked into Home Assistant directly.

**Comment by maddox** [comment]
This was fixed with version `2.3.4`.
